D.gitignore

```diff
@@ -19,6 +19,7 @@
 docs.json
 __dummy.html
 docs/
+*-test-*
```

Comment:
This doesn't really have much context - could you add a comment above to indicate what this rule covers?